src/lib.rs

```diff
@@ -37,12 +37,49 @@ use cfg_if::cfg_if;
 
 pub use self::rewriter::{
     rewrite_str, AsciiCompatibleEncoding, CommentHandler, DoctypeHandler, DocumentContentHandlers,
-    ElementContentHandlers, ElementHandler, EndHandler, EndTagHandler, HandlerResult, HtmlRewriter,
-    MemorySettings, RewriteStrSettings, Settings, TextHandler,
+    ElementContentHandlers, ElementHandler, EndHandler, EndTagHandler, HandlerNormalTypes,
```

Comment:
Let's rename `HandlerSendTypes` and `HanderNormalTypes` to `SendHandlerTypes` and `LocalHandlerTypes`